deployment/docker-compose.yml

```diff
@@ -51,13 +51,13 @@ services:
       retries: 3
 
   database_migrations: # This is the base -- the _image and _built containers are defined below
-    image: "radixdlt/babylon-ng-database-migrations:1.1.0"
```

Comment:
This should be updated to v1.2.1 as part of this release preparation I think?

Reply:
Correct. Fixed.